hummingbot/core/data_type/order_book_tracker_data_source.py

```diff
@@ -51,6 +49,13 @@ async def get_trading_pairs(self) -> List[str]:
     async def get_new_order_book(self, trading_pair: str) -> OrderBook:
         raise NotImplementedError
 
+    async def listen_for_subscriptions(self):
```

Comment:
I think this should be an abstract method, like the rest of the listeners

Reply:
No, that will force me to update all connectors that don't include the method, and I don't wan't to make such a global change as part of this PR

Reply:
I see, that makes sense